content/applications/hr/appraisals/360.rst

```diff
@@ -0,0 +1,85 @@
+============
+360 Feedback
+============
+
+Odoo's **Appraisals** app lets managers gather feedback on their direct reports from anyone in the
+company. Input from colleagues gives managers a fuller view of each employee's strengths, growth
+areas, and collaboration skills.
+
+.. _appraisals/360-dashboard:
+
+360 feedback dashboard
+======================
+
+The :guilabel:`360 Feedback` section displays information for all the surveys configured for the
+**Appraisals** app. To view the surveys and their statistics, navigate to :menuselection:`Appraisals
+app --> Configuration --> 360 Feedback`.
+
+.. image:: 360/survey-list.png
+   :alt: A list view of all available surveys in the Appraisals application.
+
+Each appraisal, or survey, is presented on its own line on the :guilabel:`360 Feedback` dashboard,
+along with various information related to that particular appraisal.
+
+Each appraisal includes the following information:
+
+- :guilabel:`Survey Name`: the name of the specific survey.
+- :guilabel:`Responsible`: the employee responsible for the survey, including the month and year
+  they were given that designation.
+- :guilabel:`Questions`: the number of questions in that particular survey.
+- :guilabel:`Average Duration`: the average time a user spends completing the survey.
+- :guilabel:`Registered`: the number of people who have been sent the survey.
+- :guilabel:`Completed`: the number of people who have completed the survey.
+- :guilabel:`Certified`: if the survey is for a certification, the percentage of users who have
+  passed the certification.
+
+Each appraisal row offers two actions:
+
+- :guilabel:`Test`: Opens the appraisal in a new tab to preview the questions without submitting
+  answers. Close the tab to return to the list, or click :icon:`oi-arrow-right` :guilabel:`Go to
+  Survey`/:guilabel:`Go to Appraisal` for the full form.
+- :guilabel:`See Results`: :ref:`View response analytics <appraisals/view>`.
+
+.. _appraisals/request:
```

Comment:
Nitpick– please update the custom anchor to be more descriptive.

Doing so keeps every anchor unique, avoiding the error "Duplicate explicit target name" when trying to use the same anchor throughout documentation
```suggestion
.. _appraisals/360-request-feedback:
```